### ISSUE TITLE
Storybook v8 compatibility

### ISSUE THREAD

**Issue by soullivaneuh** [maraisr/storybook-addon-grid#18]
# Summary

Storybook v8 is telling this plugin is incompatible on startup:

```
WARN The following packages are incompatible with Storybook 8.0.0 as they depend on different major versions of Storybook packages:
WARN - storybook-addon-grid@0.4.2
```

However the package looks to work properly. Maybe just a settings to change? :thinking: 

Ref: https://storybook.js.org/docs/addons/addon-migration-guide

**Comment by maraisr** [comment]
Hmmm has things changed again? I was fairly sure we did the work to convert things over already: https://github.com/storybookjs/storybook/issues/20529#issuecomment-1790375132

**Comment by maraisr** [comment]
Ah, yeah things changed. Ive just published version [`v0.5.0`](https://www.npmjs.com/package/storybook-addon-grid/v/0.5.0) that has the v8 support deps applied. 

Please let me know if it looks good your end. And thank you again so much for raising the issue <3
